css/alerts.css

```diff
@@ -0,0 +1,41 @@
+.alert {
+  font-size: 1rem;
+  margin: auto;
+  height: 3rem;
+  width: 100%;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+  height: max-content;
+  display: flex;
+  align-items: center;
+  border-radius: 0.4rem;
+  padding-left: 2rem;
+  gap: 0.8rem;
```

Comment:
you can give a shorthand property for padding instead of using padding-top,padding-botton,padding-left